docs/src/main/asciidoc/performance-measure.adoc

```diff
@@ -162,3 +163,51 @@ $ date +"%T.%3N" &&  ./target/quarkus-timing-runner
 --
 
 The difference between the final timestamp and the first timestamp is the total startup time for the application to serve the first request.
+
+== Additional flags applied by Quarkus
+
+When Quarkus invokes GraalVM `native-image` it will apply some additional flags by default.
+
+You might want to know about the following ones in case you're comparing performance properties with other builds.
+
+=== Disable fallback images
+
+Fallback native images are a feature of GraalVM to "fall back" to run your application in the normal JVM, should the compilation
+to native code fail for some reason.
+
+Quarkus disables this feature by setting `-H:FallbackThreshold=0`: this will ensure you get a compilation failure rather
+risking to not notice that the application is unable to really run in native mode.
+
+If you instead want to just run in Java mode, that's totally possible: just skip the native-image build and run it as a jar.
+
+=== Disable Isolates
+
+Isolates are a neat feature of GraalVM, but Quarkus isn't using them at this stage.
+
+Disable via `-H:-SpawnIsolates`.
+
+=== Disable auto-registration of all Service Loader implementations
+
+Quarkus extensions can automatically pick the right services they need, while GraalVM's native-image defaults to include
+all services it's able to find on the classpath.
+
+We prefer listing services explicitly as it produces better optimised binaries. Disable it as well by setting `-H:-UseServiceLoaderFeature`.
+
+=== Better default for Garbage Collection implementation
+
+The default in GraalVM seems meant to optimise for short lived processes. Quarkus defaults to server applications, so switch:
+set `-H:InitialCollectionPolicy=com.oracle.svm.core.genscavenge.CollectionPolicy$BySpaceAndTime`.
+
+=== Others ...
+
+This section is provided as high level guidance, but can't presume to be comprehensive as some flags are controlled
+ dynamically by the extensions, the platform you're building on, configuration details, your code and possibly
+ a combination of all these.
+
+Generally speaking the ones listed here are those most likely to affect performance metrics, but in the right
+circumstances one could observe non negligible impact from the other flags too:
```

Comment:
Really minor but did you really want a colon here at the end of the sentence?

Reply:
no I didn't. There was a sentence t hat I decided to remove.. Thanks!